Lua/Indent.tmPreferences

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <plist version="1.0">
-<dict>
-	<key>name</key>
-	<string>Indent</string>
-	<key>scope</key>
-	<string>source.lua</string>
-	<key>settings</key>
 	<dict>
-		<key>decreaseIndentPattern</key>
-		<string>^\s*(elseif|else|end|\})\s*$</string>
-		<key>increaseIndentPattern</key>
-		<string>^\s*(else|elseif|for|(local\s+)?function|if|repeat|until|while)\b((?!end).)*$|\{\s*$</string>
+		<key>name</key>
+		<string>Indent</string>
+		<key>scope</key>
+		<string>source.lua</string>
+		<key>settings</key>
+		<dict>
+			<key>decreaseIndentPattern</key>
+			<string>(^\s*\b(elsei|elseif|else|end|until)\b.*$|^((?!\{).)*\}\;?.*$)</string>
```

Comment:
Googling for "Lua elsei" doesn't return anything that looks useful. Is `elsei` supposed to be in there?

Reply:
`elsei` prevents the indentation jumping back and forth when typing `elseif`.
I doubt anyone would use elsei as a variable anyway.